src/services/measures/attitude/client/attitude.js

```diff
@@ -39,24 +99,28 @@ Gauges.Widgets.attitude = {
 					}
 				});
 			})
-			.add( this, this.publishReadyEvent )
+			.add( this, function(sequence) {
+				this.publishReadyEvent(sequence);
```

Comment:
never mind. In the end we use update only for updating the UI, so the state of the widget is unaffected anyway :)